code/datums/supplypacks/recreation_vr.dm

```diff
@@ -38,4 +38,100 @@
 			)
 	cost = 50
 	containertype = /obj/structure/closet/crate
-	containername = "wolfgirl cosplay crate"
\ No newline at end of file
+	containername = "wolfgirl cosplay crate"
+
+/datum/supply_packs/randomised/recreation/figures_vr
+	name = "Action figures crate"
+	num_contained = 5
+	contains = list(
+			/obj/item/toy/figure/agent,
+			/obj/item/toy/figure/assistant,
+			/obj/item/toy/figure/atmos,
+			/obj/item/toy/figure/bartender,
+			/obj/item/toy/figure/borg,
+			/obj/item/toy/figure/captain,
+			/obj/item/toy/figure/cargotech,
+			/obj/item/toy/figure/ce,
+			/obj/item/toy/figure/chaplain,
+			/obj/item/toy/figure/chef,
+			/obj/item/toy/figure/chemist,
+			/obj/item/toy/figure/clown,
+			/obj/item/toy/figure/cmo,
+			/obj/item/toy/figure/corgi,
+			/obj/item/toy/figure/detective,
+			/obj/item/toy/figure/dsquad,
+			/obj/item/toy/figure/engineer,
+			/obj/item/toy/figure/ert,
+			/obj/item/toy/figure/gardener,
+			/obj/item/toy/figure/geneticist,
+			/obj/item/toy/figure/hop,
+			/obj/item/toy/figure/hos,
+			/obj/item/toy/figure/janitor,
+			/obj/item/toy/figure/librarian,
+			/obj/item/toy/figure/md,
+			/obj/item/toy/figure/mime,
+			/obj/item/toy/figure/miner,
+			/obj/item/toy/figure/ninja,
+			/obj/item/toy/figure/paramedic,
+			/obj/item/toy/figure/psychologist,
+			/obj/item/toy/figure/qm,
+			/obj/item/toy/figure/rd,
+			/obj/item/toy/figure/roboticist,
+			/obj/item/toy/figure/scientist,
+			/obj/item/toy/figure/secofficer,
+			/obj/item/toy/figure/syndie,
+			/obj/item/toy/figure/virologist,
+			/obj/item/toy/figure/warden,
+			/obj/item/toy/figure/wizard,
+			/obj/item/toy/griffin,
+			/obj/item/toy/owl
+			)
+	cost = 50
```

Comment:
Should be as expensive as the collectable hats crate. Just something to burn cargo points on during a slow shift.